packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.scss

```diff
@@ -0,0 +1,320 @@
+:host {
+  position: relative;
+  display: inline-block;
+
+  & button {
+    padding: 0;
+    margin: 0;
+    cursor: pointer;
+    background: none;
+    border: none;
+    border-radius: 50%;
+
+    &:focus-visible {
+      outline-style: solid;
+      outline-color: var(--gse-ui-avatar-focus);
+      outline-offset: var(--gse-semantic-focusRing-offset);
+
+      &:has(.gux-large) {
+        outline-width: var(--gse-ui-avatar-focusRing-large-border-width);
+      }
+
+      &:has(.gux-medium) {
+        outline-width: var(--gse-ui-avatar-focusRing-medium-border-width);
+      }
+
+      &:has(.gux-small) {
+        outline-width: var(--gse-ui-avatar-focusRing-small-border-width);
+      }
+
+      &:has(.gux-xsmall) {
+        outline-width: var(--gse-ui-avatar-focusRing-xsmall-border-width);
+      }
+    }
+  }
+}
+
+.gux-avatar {
+  position: relative;
+  overflow: hidden;
+  border-radius: 50%;
+
+  &.gux-large {
+    width: var(--gse-ui-avatar-large-size);
+    height: var(--gse-ui-avatar-large-size);
+
+    & .content {
+      width: calc(100% - var(--gse-ui-avatar-large-statusRing-width) * 2);
+      height: calc(100% - var(--gse-ui-avatar-large-statusRing-width) * 2);
+      margin: var(--gse-ui-avatar-large-statusRing-width);
+      font-size: var(--gse-ui-avatar-large-initials-fontSize);
+      border-width: var(--gse-ui-avatar-content-large-border-width);
+    }
+  }
+
+  &.gux-medium {
+    width: var(--gse-ui-avatar-medium-size);
+    height: var(--gse-ui-avatar-medium-size);
+
+    & .content {
+      width: calc(100% - var(--gse-ui-avatar-medium-statusRing-width) * 2);
+      height: calc(100% - var(--gse-ui-avatar-medium-statusRing-width) * 2);
+      margin: var(--gse-ui-avatar-medium-statusRing-width);
+      font-size: var(--gse-ui-avatar-medium-initials-fontSize);
+    }
+  }
+
+  &.gux-small {
+    width: var(--gse-ui-avatar-small-size);
+    height: var(--gse-ui-avatar-small-size);
+
+    & .content {
+      width: calc(100% - var(--gse-ui-avatar-small-statusRing-width) * 2);
+      height: calc(100% - var(--gse-ui-avatar-small-statusRing-width) * 2);
+      margin: var(--gse-ui-avatar-small-statusRing-width);
```

Comment:
Just wanted to confirm `--gse-ui-avatar-small-statusRing-width` was the intended token here since this is a margin style.

Reply:
@jason-evans-genesys Yes, it's applying a width to a border around an element by adding margin & exposing the background. I thought this was an easier way to handle widths & existing borders.